app/mailers/emailer.rb

```diff
@@ -80,7 +80,6 @@ def welcome_to_new_user_added_by_affiliate(affiliate, user, current_user)
     @added_user_email = user.email
     @affiliate_display_name = affiliate.display_name
     @affiliate_site_url = site_url(affiliate)
-    @complete_registration_url = site_url(affiliate)
```

Comment:
we are not completing registrations anymore. This was left over from when a user account had to be validated. Login.gov handles this part.